src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -216,38 +219,38 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
             string filterText)
         {
             var helper = CompletionHelper.GetHelper(document);
-            return FilterItems(helper, items, filterText);
+            var itemsWithPatternMatch = items.Select(
+                item => (item, helper.GetMatch(item.FilterText, filterText, includeMatchSpans: false, CultureInfo.CurrentCulture)));
+
+            var builder = ImmutableArray.CreateBuilder<CompletionItem>();
+            FilterItems(helper, itemsWithPatternMatch, filterText, builder);
+            return builder.MoveToImmutable();
```

Comment:
doc how you know MoveToImmutable is safe here.

Reply:
😬

Reply:
Fixed.